src/egon/data/datasets/industrial_gas_demand.py

```diff
@@ -23,23 +23,32 @@ def __init__(self, dependencies):
         )
 
 
-def download_CH4_industrial_demand():
+def download_CH4_industrial_demand(scn_name="eGon2035"):
     """Download the CH4 industrial demand in Germany from the FfE open data portal
-    
+
+    Parameters
+    ----------
+    scn_name : str
+        Name of the scenario
+
     Returns
     -------
-    CH4_industrial_demand : 
+    CH4_industrial_demand :
         Dataframe containing the CH4 industrial demand in Germany
-        
+
     """
     # Download the data and the id_region correspondance table
     correspondance_url = (
         "http://opendata.ffe.de:3000/region?id_region_type=eq.38"
     )
     url = "http://opendata.ffe.de:3000/opendata?id_opendata=eq.66&&year=eq."
 
+    if scn_name == 'eGon2035':
+        year = "2035"
+    else:
+        year = "2050"
+

```

Comment:
You can select the year from scenario_parameters (`get_sector_parameters("global", scn_name)["population_year"]`)

Reply:
I run your branch in test mode and it failed due to the datatype of `year`. `get_sector_parameters` returns an `int` value, but in line 51 a `str` value is needed.

Reply:
Thank you for letting me know, fixed by 72cc1f7